code/modules/admin/player_panel.dm

```diff
@@ -1,676 +1,4 @@
-/datum/admins/proc/player_panel_new()//The new one

```

Comment:
Another in a good example of SHIT THAT SHOULD BE TGUI BECAUSE THIS HTML SHIT IS UGLY